src/sounds.c

```diff
@@ -314,24 +314,35 @@ void update_player_sounds(void)
     }
     if (game.play_gameturn != 0)
     {
-        // Rare message easter egg
-        if ((game.play_gameturn % 20000) == 0)
+        // Easter Egg Speeches
+
+        // Interval for easter egg speeches. Original DK value was 20000 (16.6 minutes)
+        if (game.conf.rules.game.easter_egg_speech_interval != 0 && (game.play_gameturn % game.conf.rules.game.easter_egg_speech_interval) == 0)
```

Comment:
No. This does prevent the crash.

If a condition returns false in the first part it will not check the rest of the condition. This is common in programming.